Buildings/Templates/ChilledWaterPlant/BaseClasses/PartialChilledWaterLoop.mo

```diff
@@ -0,0 +1,205 @@
+      final m2_flow_nominal=mPri_flow_nominal)
+    "Chilled water return section"
+    annotation (Placement(transformation(
+      extent={{10,-10},{-10,10}},rotation=90,origin={-40,-72})));
+  inner replaceable
+    Buildings.Templates.ChilledWaterPlant.Components.PrimaryPumpGroup.Headered
+    pumPri constrainedby
+    Buildings.Templates.ChilledWaterPlant.Components.PrimaryPumpGroup.Interfaces.PrimaryPumpGroup(
+      redeclare final package Medium = MediumCHW,
+      final mTot_flow_nominal=mPri_flow_nominal,
+      final dp_nominal=dpPri_nominal,
+      final nChi=nChi,
+      final has_ParChi=chiGro.typ == Buildings.Templates.ChilledWaterPlant.Components.Types.ChillerGroup.ChillerParallel,
+      final has_byp=has_byp,
+      final has_WSEByp=has_WSEByp,
+      final has_comLeg=not pumSec.is_none)
+    "Chilled water primary pump group"
+    annotation (Placement(transformation(extent={{0,0},{20,20}})));
+  inner replaceable
+    Buildings.Templates.ChilledWaterPlant.Components.SecondaryPumpGroup.None
+    pumSec constrainedby
+    Buildings.Templates.ChilledWaterPlant.Components.SecondaryPumpGroup.Interfaces.SecondaryPumpGroup(
+      redeclare final package Medium = MediumCHW,
+      final mTot_flow_nominal=mSec_flow_nominal,
+      final dp_nominal=dpSec_nominal)
+    "Chilled water secondary pump group"
+    annotation (Placement(transformation(extent={{60,0},{80,20}})));
+
+  inner replaceable Components.Controls.OpenLoop con
+    constrainedby
+    Buildings.Templates.ChilledWaterPlant.Components.Controls.Interfaces.PartialController(
+      final nChi=nChi,
+      final nPumPri=nPumPri,
+      final nPumSec=nPumSec,
+      final is_airCoo=is_airCoo)
+    annotation (Placement(
+      transformation(
+      extent={{10,10},{-10,-10}},
+      rotation=180,
+      origin={70,60})));
+
+  Buildings.Templates.Components.Sensors.Temperature TCHWRet(
+    redeclare final package Medium = MediumCHW,
+    final have_sen=true,
+    final typ=Buildings.Templates.Components.Types.SensorTemperature.InWell,
+    final m_flow_nominal=mSec_flow_nominal)
+    "Chilled water return temperature"
+    annotation (Placement(transformation(extent={{120,-80},{140,-60}})));
+  Buildings.Templates.Components.Sensors.DifferentialPressure dpCHW(
+    redeclare final package Medium = MediumCHW,
+    final have_sen=true,
+    m_flow_nominal=0)
+    "Chilled water demand side differential pressure"
+    annotation (Placement(transformation(extent={{10,10},{-10,-10}},
+        rotation=90,
+        origin={170,-30})));
+  Buildings.Templates.Components.Sensors.Temperature TCHWSup(
+    redeclare final package Medium = MediumCHW,
+    final have_sen=true,
+    final typ=Buildings.Templates.Components.Types.SensorTemperature.InWell,
+    final m_flow_nominal=mSec_flow_nominal)
+    "Chilled water supply temperature"
+    annotation (Placement(transformation(extent={{120,0},{140,20}})));
+  Buildings.Templates.Components.Sensors.Temperature TCHWRetByp(
```

Comment:
Let's discuss the nomenclature I proposed at https://docs.google.com/document/d/1LeutsY9__ClaIEjmvRHkIAMGX5dlW0Xo1BFGvJTHjs4/edit
(For consistency with the VAV template, this should be named `TCHWPlaRet` rather than `TPlaCHWRet`.)